src/FSharp.Control.TaskSeq/TaskSeq.fsi

```diff
@@ -1177,8 +1179,8 @@ type TaskSeq =
 
     /// <summary>
     /// Applies the asynchronous function <paramref name="folder" /> to each element in the task sequence, threading an accumulator
-    /// argument of type <typeref name="'State" /> through the computation.  If the input function is <paramref name="f" /> and the elements are <paramref name="i0...iN" />
-    /// then computes <paramref name="f (... (f s i0)...) iN" />.
+    /// argument of type <typeref name="'State" /> through the computation.  If the input function is <code>f</code> and the elements are <code>i0...iN</code>
+    /// then computes <code>f (... (f s i0)...) iN</code>.
```

Comment:
And here:
```suggestion
    /// argument of type <typeref name="'State" /> through the computation.  If the input function is <paramref name="f" /> and the elements are <paramref name="i0...iN" />
    /// then computes <paramref name="f (... (f s i0)...) iN" />.
```

Reply:
👍  now paramref as discussed
(also I meant `<c>` anyway)